parameter.go

```diff
@@ -0,0 +1,230 @@
+package tfe
```

Comment:
I would call this file `policy_set_parameter.go` to make it explicit what it is. Only `parameter` seems very generic.